jsx/Card.js

```diff
@@ -37,9 +36,9 @@ class Card extends Component {
   }
 
   /**
-   * Render the React component
+   * Renders the React component.
    *
-   * @return {object}
+   * @return {string} - HTML markup for the component
```

Comment:
`@return {react}` might be better here instead of string?

Reply:
I thought about that too. I'm fine with either your suggestion or @return {JSX}.

Reply:
I didn't think about the other two possibilities. Whatever you think is best will be great!